### ISSUE TITLE
Mail script command

### ISSUE THREAD

**Issue by RadianFord** [rathena/rathena#3880]
* **rAthena Hash**: 

* **Client Date**: 

* **Server Mode**: Pre-renewal

* **Description of Issue**: Use this script https://rathena.org/board/topic/117956-rolling-in-the-deep/?do=findComment&comment=356321 as base of example and select an item that has refine on it wait until the npc return the item when it got cancelled. refine is gone.

**Comment by Lemongrass3110** [comment]
This is more or less a feature request.

As you can see in script_commands.txt the mail command does not feature a parameter for refine.
```
*mail <destination id>,"<sender name>","<title>","<body>"{,<zeny>{,<item id array>,<item amount array>{,<item card0 array>{,<item card1 array>{,<item card2 array>{,<item card3 array>
		{,<random option id0 array>, <random option value0 array>, <random option paramter0 array>{,<random option id1 array>, <random option value1 array>, <random option paramter1 array>
		{,<random option id2 array>, <random option value2 array>, <random option paramter2 array>{,<random option id3 array>, <random option value3 array>, <random option paramter3 array>
		{,<random option id4 array>, <random option value4 array>, <random option paramter4 array>}}}}}}};
```

**Comment by Lemongrass3110** [comment]
Fixed in e5c22b3
